.github/workflows/code-checks.yml

```
@@ -34,7 +34,9 @@ jobs:
       run: |
         pip install torch==1.11 --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
         python ./requirements/pytorch/adjust-versions.py requirements/pytorch/extra.txt
+        # todo: adjust requirements for both code-bases
         pip install -r requirements/pytorch/devel.txt --find-links https://download.pytorch.org/whl/cpu/torch_stable.html
+        pip install "mypy==0.920"
```

The correct pin in master is:

```suggestion
        pip install "mypy==0.971"
```